src/main/java/com/smartcar/sdk/data/BatchResponse.java

```diff
@@ -121,6 +121,20 @@ public void setRequestId(String requestId) {
         return get("/battery", VehicleBattery.class);
     }
 
+    /**
+     * Get response from the battery/capacity endpoint
+     *
+     * @return the battery capacity of the vehicle
+     *
+     * @throws BatchResponseMissingException if this endpoint was not
+     * part of the batch response
+     * @throws SmartcarException if the request for this endpoint returned an
+     * HTTP error code
+     */
+    public SmartcarResponse<VehicleBatteryCapacity> batteryCapacity() throws BatchResponseMissingException, SmartcarException {
+        return get("battery/capacity", VehicleBatteryCapacity.class);
```

Comment:
```suggestion
        return get("/battery/capacity", VehicleBatteryCapacity.class);
```

Reply:
Found this by running the SDK locally using: 

`main/java/com/smartcar/sdk/Request.java` (new file)
```java
import com.smartcar.sdk.AuthClient;
import com.smartcar.sdk.Vehicle;
import com.smartcar.sdk.data.SmartcarResponse;
import com.smartcar.sdk.data.VehicleIds;

public class Request {
  public static void main(String[] args) throws Exception {
    String accessToken = "TOKEN";
    SmartcarResponse<VehicleIds> res = AuthClient.getVehicleIds(accessToken);
    String vid = res.getData().getVehicleIds()[2];
    Vehicle vehicle = new Vehicle(vid, accessToken);
    System.out.println(vehicle.batch(new String[] {"/battery/capacity"}).batteryCapacity());
  }
}
```

`build.gradle` (addition)
```gradle
// Run the request with: ./gradlew request
task('request', type: JavaExec) {
  classpath = sourceSets.main.runtimeClasspath
  main = 'Request'
}
```

execute using: `./gradlew request`